src/PyCall.jl

```diff
@@ -271,22 +271,22 @@ end
 # computing hashes of PyObjects
 
 const pysalt = hash("PyCall.PyObject") # "salt" to mix in to PyObject hashes
-hashsalt(x) = hash(x, pysalt)
 
-function hash(o::PyObject)
+function hash(o::PyObject, salt::UInt=zero(UInt))
```

Comment:
```suggestion
function hash(o::PyObject, salt::UInt)
```
The default value is not needed here because Julia defines `hash(x::Any) = hash(x, zero(UInt))` for all types.